src/Input.js

```diff
@@ -45,32 +47,34 @@ class Input extends Component {
         }
       })
       this.props.setFilteredMovies(filteredMovies)
-    } else {
-      // this.props.refresh();
     }
   }
 
+  returnToMovies = () => {
```

Comment:
Can we get rid of this function now that our backspacing returns us to all movies?